stable/aws-for-fluent-bit/templates/configmap.yaml

```diff
@@ -241,33 +241,33 @@ data:
 
 {{- if .Values.kinesis_streams.enabled }}
     [OUTPUT]
-        Name            kinesis_streams
-        Match           {{ .Values.kinesis_streams.match }}
-        region          {{ .Values.kinesis_streams.region }}
-        stream          {{ .Values.kinesis_streams.stream }}
+        Name                kinesis_streams
+        Match               {{ .Values.kinesis_streams.match }}
+        region              {{ .Values.kinesis_streams.region }}
+        stream              {{ .Values.kinesis_streams.stream }}
         {{- if .Values.kinesis_streams.role_arn }}
-        role_arn        {{ .Values.kinesis_streams.role_arn }}
+        role_arn            {{ .Values.kinesis_streams.role_arn }}
         {{- end }}
         {{- if .Values.kinesis_streams.endpoint }}
-        endpoint        {{ .Values.kinesis_streams.endpoint }}
+        endpoint            {{ .Values.kinesis_streams.endpoint }}
         {{- end }}
         {{- if .Values.kinesis_streams.sts_endpoint }}
-        sts_endpoint    {{ .Values.kinesis_streams.sts_endpoint }}
+        sts_endpoint        {{ .Values.kinesis_streams.sts_endpoint }}
         {{- end }}
         {{- if .Values.kinesis_streams.time_key }}
-        time_key        {{ .Values.kinesis_streams.time_key }}
+        time_key            {{ .Values.kinesis_streams.time_key }}
         {{- end }}
         {{- if .Values.kinesis_streams.time_key_format }}
-        time_key_format {{ .Values.kinesis_streams.time_key_format }}
+        time_key_format     {{ .Values.kinesis_streams.time_key_format }}
         {{- end }}
         {{- if .Values.kinesis_streams.external_id }}
-        time_key_format {{ .Values.kinesis_streams.external_id }}
+        external_id         {{ .Values.kinesis_streams.external_id }}
         {{- end }}
         {{- if .Values.kinesis_streams.auto_retry_requests }}
-        time_key_format {{ .Values.kinesis_streams.auto_retry_requests }}
+        auto_retry_requests {{ .Values.kinesis_streams.auto_retry_requests }}
```

Comment:
can you please add support for time_key_format as well

Reply:
It seems that time_key_format is already supported. If I am mistaken, please let me know.
* https://github.com/aws/eks-charts/blob/master/stable/aws-for-fluent-bit/templates/configmap.yaml#L260-L262